docs/introduction.rst

```diff
@@ -116,4 +116,25 @@ Other benefits for developers include:
   dependencies, which can be used by applications, and which can take a lot
   of the work out of application development.
 
+
+In general Flatpak is best suited suited for desktop applications and
+while command line applications also work, it may not be suited in some
+cases:
+
+- Applications needs to elevate priviledges using ``su, sudo, pkexec``
+  etc. or needs to call ``setcap`` and set priviledges. Flatpak cannot
+  run in SUID binaries inside the sandbox.
+- Application needs to read ``/proc`` from host or have unfiltered access
+  to processes. This is not allowed as Flatpak has a private proc.
+- Application uses a syscall that is blocklisted by Flatpak's seccomp
+  filter. For example, Flatpak won't allow spawning sub-namespaces
+  in the sandbox.
+- Kernel modules, drivers, daemons are non application packages and
+  won't work inside a Flatpak.
+- Applications that need systemd services or udev rules. Flatpak does not
```

Comment:
We do have applications that ask you to setup `udev` on the host and then work. Not sure if this makes it sound, like it's not working at all.

Wondering if this should separate hard errors/requirements (no sudo etc) and unconvinient stuff?

Reply:
Yea I can change that a bit. udev rules won't work from inside the sandbox.